tests/unit/test_request.py

```diff
@@ -0,0 +1,21 @@
+import unittest
```

Comment:
remove unittest

Reply:
if i remove unittest than we cant call directly 
```
if __name__ == '__main__':
    unittest.main()```

Reply:
remove that also, we are using `pytest` currently

Reply:
done